### PR TITLE
RATEST-130:Create A Feature File For EditPatientRelationship and Steps

### PR DESCRIPTION
**Pull request by sherrif10**
https://issues.openmrs.org/browse/RATEST-130
 cc @kaweesi  kindly review thanks

### CODE REVIEW

File: src/test/java/org/openmrs/contrib/qaframework/automation/EditPatientRelationshipSteps.java [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+	}
+
+	@Then("System loads clicianFacingPatientDashboardPage")
+	public void loadClinicianFacingPatientDashboardPage() {
+		matchPatientIds(firstPatientIdentifier);
+
+	}
+
+	@After(RunTest.HOOK.SELENIUM_DASHBOARD)
+	public void destroy() {
```
Comment (kdaud) [inline src/test/java/org/openmrs/contrib/qaframework/automation/EditPatientRelationshipSteps.java:31]: Move this function above the previous one for better code workflow

File: src/test/java/org/openmrs/contrib/qaframework/automation/EditPatientRelationshipSteps.java [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+		quit();
+	}
+
+	@Then("User clicks on editRegistration")
+	public void clickEditPatientRegistration() {
```
Comment (kdaud) [inline src/test/java/org/openmrs/contrib/qaframework/automation/EditPatientRelationshipSteps.java:36]: Was it an oversight to skip the function implementation

File: src/test/java/org/openmrs/contrib/qaframework/automation/EditPatientRelationshipSteps.java [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+
+	}
+	@Then("User clicks on EditRelationship")
+	public void clickEditRelationship() {
```
Comment (kdaud) [inline src/test/java/org/openmrs/contrib/qaframework/automation/EditPatientRelationshipSteps.java:40]: same here

File: src/test/java/org/openmrs/contrib/qaframework/automation/EditPatientRelationshipSteps.java [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+
+	}
+
+	@And("User click on find Relationshiptype")
+	public void findRelationshipType() {
```
Comment (kdaud) [inline src/test/java/org/openmrs/contrib/qaframework/automation/EditPatientRelationshipSteps.java:45]: And here

File: src/test/resources/features/refapp/06-patient/EditPatientRelationship.feature [hunk 5]
```diff
@@ -0,0 +1,2 @@
+Feature: EditRelationshipType
```
Comment (kdaud) [inline src/test/resources/features/refapp/06-patient/EditPatientRelationship.feature:1]: Did you just forget to write the user scenarios in the file

**Comment by kdaud** [comment]
Thanks @sherrif10 for the work. Please catch the comments above

File: src/test/resources/features/refapp/06-patient/EditPatientRelationship.feature (hunk 5 above)
Comment (sherrif10) [reply src/test/resources/features/refapp/06-patient/EditPatientRelationship.feature:1]: This is work in Progress thanks

**Comment by sherrif10** [comment]
Thanks for your review @kdaud

File: src/test/java/org/openmrs/contrib/qaframework/automation/EditPatientRelationshipSteps.java [hunk 6]
```diff
@@ -0,0 +1,51 @@
...
+	}
+
+	@After(RunTest.HOOK.SELENIUM_DASHBOARD)
+	public void destroy() {
+		quit();
+	}
+
+	@Given("System loads clinicianFacingPatientDashboardPage")
+	public void loadClinicianFacingPatientDashboardPage() throws Exception {
+
```
[the remark this answers is not in the document]
Comment (sherrif10) [reply src/test/java/org/openmrs/contrib/qaframework/automation/EditPatientRelationshipSteps.java:31]: Thanks the only challenge am having at the moment is that ,  i want to be able to use some methods that can are inherited by page class in distro

File: src/test/resources/features/refapp/06-patient/EditPatientRelationship.feature [hunk 7]
```diff
@@ -0,0 +1,16 @@
...
+
+  Scenario Outline editPatientRelationship:
```
Comment (kaweesi) [inline src/test/resources/features/refapp/06-patient/EditPatientRelationship.feature:3]: there's a syntax issue here, check existing features to know how to use scenario outline

File: src/test/java/org/openmrs/contrib/qaframework/automation/EditPatientRelationshipSteps.java (hunk 1 above)
Comment (kaweesi) [reply src/test/java/org/openmrs/contrib/qaframework/automation/EditPatientRelationshipSteps.java:25]: i always prefer After after Before and this looks alright to me a position

Comment (sherrif10) [reply src/test/java/org/openmrs/contrib/qaframework/automation/EditPatientRelationshipSteps.java:25]: Thanks for the review

File: src/test/resources/features/refapp/06-patient/EditPatientRelationship.feature (hunk 7 above)
Comment (sherrif10) [reply src/test/resources/features/refapp/06-patient/EditPatientRelationship.feature:3]: Resolved thanks , is this good to you now

**Pull request closed without merging**